Cargo.toml

```diff
@@ -9,30 +9,54 @@ repository = "https://github.com/collinsmuriuki/mpesa-rust"
 readme = "./README.md"
 license = "MIT"
 
+[features]
+default = [
+    "account_balance",
+    "b2b",
+    "b2c",
+    "bill_manager",
+    "c2b_register",
+    "c2b_simulate",
+    "express_request",
+    "transaction_reversal",
+    "transaction_status",
+]
+account_balance = []
+b2b = ["dep:base64", "dep:x509-parser", "dep:rsa", "dep:rand"]
+b2c = ["dep:base64", "dep:x509-parser", "dep:rsa", "dep:rand"]
+bill_manager = ["dep:chrono"]
+c2b_register = []
+c2b_simulate = []
+express_request = ["dep:chrono", "dep:base64"]
+transaction_reversal = ["dep:base64", "dep:x509-parser", "dep:rsa", "dep:rand"]
+transaction_status = ["dep:base64", "dep:x509-parser", "dep:rsa", "dep:rand"]
+
+
 [dependencies]
-chrono = {version = "0.4", optional = true, default-features = false, features = ["clock", "serde"] }
-openssl = {version = "0.10", optional = true}
-reqwest = {version = "0.11", features = ["json"]}
-secrecy = "0.8.0"
-serde = {version="1.0", features= ["derive"]}
+base64 = { version = "0.21", optional = true }
+chrono = { version = "0.4", optional = true, default-features = false, features = [
+    "clock",
+    "serde",
+] }
+rand = { version = "0.8", optional = true, default-features = false, features = [
+    "std",
+] }
+reqwest = { version = "0.11", features = ["json"] }
+rsa = { version = "0.9", optional = true }
+secrecy = "0.8"
+serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_repr = "0.1"
-thiserror = "1.0.37"
-wiremock = "0.5"
+thiserror = "1.0"
+x509-parser = { version = "0.15", optional = true }
+
 
 [dev-dependencies]
 dotenv = "0.15"
-tokio = {version = "1", features = ["rt", "rt-multi-thread", "macros"]}
+tokio = { version = "1", features = ["rt", "rt-multi-thread", "macros"] }
 wiremock = "0.5"
 
-[features]
-default = ["account_balance", "b2b", "b2c", "bill_manager", "c2b_register", "c2b_simulate", "express_request", "transaction_reversal", "transaction_status"]
-account_balance = ["dep:openssl"]
-b2b = ["dep:openssl"]
-b2c = ["dep:openssl"]
-bill_manager = ["dep:chrono"]
-c2b_register = []
-c2b_simulate = []
-express_request = ["dep:chrono"]
-transaction_reversal = ["dep:openssl"]
-transaction_status= ["dep:openssl"]
+
+# Support Wasm
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
Well, no need for this then

https://github.com/c12i/mpesa-rust/pull/83/files#r1471106283